doc/themes/scikit-learn/static/js/sphinx_copybutton.js

```diff
@@ -0,0 +1,116 @@
+// Please see details at https://github.com/choldgraf/sphinx-copybutton
```

Comment:
This is not sufficient according to https://github.com/choldgraf/sphinx-copybutton/blob/master/LICENSE:

"
The above copyright notice and this permission notice shall be included in all
copies or substantial portions of the Software.
"